_data/Jawad-Saqib/Jawad-Saqib.md

```diff
@@ -0,0 +1,7 @@
+---
+name: JAWAD SAQIB BHATTI
+institution: COMSATS University Islamabad 🚩
+profile_pic: jawad.jpg
+quote: Always be consistent, do something of your interest and gain skill to excel in the field. Also never stop learning :)
```

Comment:
Looks like this quote is 117 characters in length. README.md states that this can not exceed 100 characters. Please revise this and we can get you squared away!

From README.md:
> name: FULLNAME-OR-NICKNAME # No longer than 28 characters
institution: INSTITUTION-NAME 🚩 # no longer than 58 characters
> profile_pic: mona.png # Name and extension of your profile picture(ex. mona.png) The picture must be squared and 544px on width and height.
> quote: YOUR-SENIOR-QUOTE # **no longer than 100 characters**, avoid using quotes(") to guarantee the format remains the same.
> github_user: YOUR-GITHUB-USERNAME